keystore/keyring.go

```diff
@@ -29,19 +29,40 @@ var TestKeyRing *TestKeyRingHolder
 
 // TestKeyStore is a struct that holds a Keystore of all the test keys
 type TestKeyRingHolder struct {
-	EthereumKeys   KeyRing
-	CentrifugeKeys KeyRing
+	EthereumKeys   ETHKeyRing
+	CentrifugeKeys SUBKeyRing
 }
 
 // KeyRing holds the keypair related to a specfic keypair type
 type KeyRing map[string]crypto.Keypair
 
+type ETHKeyRing map[string]*secp256k1.Keypair
+type SUBKeyRing map[string]*sr25519.Keypair
```

Comment:
I don't really seen any benefit to the additional types as the keys themselves are typed.

Reply:
I've removed them and changed the Keyring to the map types. If this is not what you meant, let me know.